packages/css-data/src/property-parsers/translate.test.ts

```diff
@@ -0,0 +1,81 @@
+import { test, expect } from "@jest/globals";
+import { parseTranslate } from "./translate";
+
+test("parses a valid translate value", () => {
```

Comment:
I would just write all tests in parse-css-value.test.ts
Btw I'm trying to avoid snapshots for json structures to make formatting more compact.

Reply:
sure will update it 👍